ansible/config_sonic_basedon_testbed.yml

```diff
@@ -414,15 +414,30 @@
     when: proxy_env is defined and deploy is defined and deploy|bool == true
 
   - block:
+      - name: Load variables from topology file for topo_{{ topo }}.yml
+        include_vars: "vars/topo_{{ topo }}.yml"
+
       - name: saved original minigraph file in SONiC DUT(ignore errors when file does not exist)
         shell: mv /etc/sonic/minigraph.xml /etc/sonic/minigraph.xml.orig
         become: true
         ignore_errors: true
+        when: dut_cfg_file_loc is not defined
```

Comment:
IMO, we should not include information like `dut_cfg_file_loc` in topology definition file. It would be better to use separate files to track the detailed DUT/VM configuration dependent information.

Reply:
For normal case, the configuration is derived from topo file. It is better to have initial configurations associated each test topo file, which could be either based on minigraph or based on a given set of config_db json files.